tools/c7n_azure/c7n_azure/actions.py

```diff
@@ -32,7 +32,29 @@
 from c7n.utils import type_schema
 
 
-class Tag(BaseAction):
+class AzureBaseAction(BaseAction):
```

Comment:
Set the metaclass `@six.add_metaclass(ABCMeta)`, because we don't really want this class instantiated.

I think you can also use multiple inheritance syntax here: AzureBaseAction(BaseAction, ABC)... but I haven't tried it.

Reply:
The whole class isn't abstract though, just one method.

Reply:
Done!